website/docs/guides/2.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,348 @@
+---
+layout: "azurerm"
+page_title: "Azure Active Directory: 2.0 Upgrade Guide"
+sidebar_current: "docs-azurerm-guide-2.0-upgrade"
+description: | -
+    This page documents how to migrate between two resources in the Azure Provider which have been renamed.
+
+---
+
+# v2.0 of the AzureRM Provider
+
+Terraform initially shipped support for the AzureRM Provider back in December 2015.
+
+Since then we've added support for 191 Resources, 58 Data Sources and have launched a couple of related Providers in the form of [the Azure Active Directory Provider](https://www.terraform.io/docs/providers/azuread/index.html) and [the Azure Stack Provider](https://www.terraform.io/docs/providers/azurestack/index.html).
+
+Version 2.0 of the AzureRM Provider is a major release and as such includes some larger-scale changes which are outlined in this document.
+
+-> **NOTE:** This guide is a Work In Progress and as such fields may be added to this guide until version 2.0 of the AzureRM Provider is released.
+
+At a high level, we're intending for version 2.0 to include the following changes:
+
+* Existing Resources will be required to be imported
```

Comment:
We might want to clarify this a bit as I read this as "I have to do something to every one of my existing Terraform AzureRM resources?":

```suggestion
* When adding pre-existing AzureRM infrastructure as new resources in Terraform, importing those Terraform resources will be required
```

Reply:
What about:

> Changes when importing existing resources - when bringing Resources provisioned outside of Terraform under Terraform's control, these resources will now be required to be imported.

Reply:
Sounds better to me